bin/plugin/commands/test/__snapshots__/changelog.js.snap

```diff
@@ -207,3 +209,15 @@ exports[`formatChangelog verify that the changelog is properly formatted 1`] = `
 
 "
 `;
+
+exports[`formatContributors verify that the contributors list is properly formatted 1`] = `
+"## First time contributors
+
+The following PRs were merged by first time contrbutors:
+
+- @kapilpaul: Add spaces in add_theme_support documentation code. ([33796](https://github.com/WordPress/gutenberg/pull/33796))
```

Comment:
Nit: this list could also be sorted by the username 😃

Reply:
Yes! Everyone loves sorting 👍

Reply:
✅